app/views/dashboard/_discovery_widget_host.html.erb

```diff
@@ -0,0 +1,4 @@
+<td><%= discovery_status_icon(host) %>&nbsp;<%= link_to trunc_with_tooltip(h(host.name)), discovered_host_path(host) %></td>
```

Comment:
Not sure how to do that, can you give me an example? The `&nbsp;` prevents the icon from getting separated from the hostname, is there a better way to do that?